assets/css/style.css

```diff
@@ -339,3 +352,265 @@ fieldset > textarea:focus {
   height: 2em;
   margin: 2em 0 1em 0;
 }
+
+@media (min-width: 768px) {
+  .nav-bar {
+    background-image: none;
+    justify-content: center;
+  }
+
+  #hamburguer {
+    display: none;
+  }
+
+  #menu-wrapper {
+    display: flex;
+  }
+
+  #menu-wrapper > a {
+    margin-right: 50px;
+    text-decoration: none;
+    color: #1a2236;
+  }
+
+  #menu-wrapper > a:hover {
+    text-decoration: underline #1a2236 2px;
+  }
+
+  .headline > .text-wrapper {
+    max-width: 55%;
+    height: 100%;
+    padding: 0;
+    margin: 0;
+    background-image: url('/assets/img/image_dots.png');
+    background-repeat: no-repeat;
+    background-size: 20vh;
+    background-position: bottom right;
+  }
+
+  .headline > .text-wrapper > p {
+    margin-right: 35%;
+  }
+
+  .button {
+    width: 15em;
+    padding: 1.3em;
+  }
+
+  .headline {
+    height: 100vh;
+    justify-content: center;
+    padding: 0 10%;
+    background-image: url('/assets/img/image_geometry_3.png');
+    background-size: 45% 100vh;
+    background-position: right;
+  }
+
+  h1 {
+    font-size: 8em;
+    margin: 0;
+  }
+
+  h2 {
+    font-size: 8em;
+  }
+
+  .projects {
```

Comment:
- [ ] Kindly update project title to make it match Figma, as you can see below, `font-size` and `font-weight` are different

## Your design:
![image](https://user-images.githubusercontent.com/50721479/187005996-fb77f708-563e-4d11-b586-43daec3435a7.png)

## Figma design:
![image](https://user-images.githubusercontent.com/50721479/187006012-081fc660-6cdd-4dbc-a932-1f6174725a20.png)